### PR TITLE
kfac recipe

### PR DESCRIPTION
**Pull request by npanpaliya**

### CODE REVIEW

File: conda-recipes/kfac-feedstock/.ci_support/linux_noarch.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+c_compiler:
+- gcc
+c_compiler_version:
+- '8'
+channel_sources:
+- conda-forge,defaults
```
Comment (jayfurmanek) [inline conda-recipes/kfac-feedstock/.ci_support/linux_noarch.yaml]: I think this doesn't need conda-forge here.

Also, I think you can  put the WML CE channel here as well (first) - just the full URL like:

```
channel_sources:
  - https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda/,defaults
```

Comment (jayfurmanek) [reply conda-recipes/kfac-feedstock/.ci_support/linux_noarch.yaml]: This should help the "clobber" warnings in the test part of the build.

File: conda-recipes/kfac-feedstock/recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - six
+    - numpy
+    - tensorflow-probability
+    - tensorflow-gpu >=1.14
```
Comment (jayfurmanek) [inline conda-recipes/kfac-feedstock/recipe/meta.yaml]: Is KFAC GPU only?

Comment (npanpaliya) [reply conda-recipes/kfac-feedstock/recipe/meta.yaml]: This line says so https://github.com/tensorflow/kfac/blob/master/setup.py#L39. However, in the README, even cpu version is mentioned. I'll replace `tensorflow-gpu` with `tensorflow`.

**Comment by npanpaliya** [comment]
Thanks @jayfurmanek . Addressed your comments. Please let me know your feedback.

**Comment by jayfurmanek** [comment]
https://github.com/IBM/powerai/issues/43

**Pull request merged**